examples/reminders/lib/feature_reminder_list/ui_components/scroll_to_index_widget.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_sticky_header/flutter_sticky_header.dart';
+import 'package:rx_bloc_list/models.dart';
+import 'package:scroll_pos/scroll_pos.dart';
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_reminder_tile.dart';
+import '../../base/common_ui_components/app_sticky_header.dart';
+import '../../base/models/reminder/reminder_model.dart';
+
+class ScrollPosWidget extends StatefulWidget {
+  const ScrollPosWidget(this.list, this.id, {Key? key}) : super(key: key);
+
+  final PaginatedList<ReminderModel> list;
+  final String id;
+
+  @override
+  State<ScrollPosWidget> createState() => _ScrollPosWidgetState();
+}
+
+class _ScrollPosWidgetState extends State<ScrollPosWidget> {
+  late final ScrollPosController _controller;
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  void initState() {
+    _controller = ScrollPosController(itemCount: widget.list.length);
+    final _correctIndex =
+        widget.list.indexWhere((element) => element.id == widget.id);
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
+        _scrollToIndex(_correctIndex);
```

Comment:
Usually `addPostFrameCallback ` goes hand by hand with https://api.flutter.dev/flutter/widgets/State/didChangeDependencies.html

Reply:
The method `didUpdateWidget` was added the `addPostFrameCallback ` was removed